changelog/fragments/bump-kb.yaml

```diff
@@ -0,0 +1,50 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bump kubebuilder commit in Operator-SDK. This fixes a bug where running `make test` 
+      would error because of Kubebuilder assets not being available at the right location. 
+      For more details, refer:
+      https://github.com/kubernetes-sigs/kubebuilder/pull/2272.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: true
```

Comment:
This isn't breaking because upgrading to the next `operator-sdk` version won't cause issues.